tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraLongLivedTokenSetMethodContract.scala

```diff
@@ -633,4 +633,322 @@ trait LinagoraLongLivedTokenSetMethodContract {
+         |      "accountId": "$ACCOUNT_ID",
+         |      "destroy": ["${LongLivedTokenId.generate.value}"]
+         |    }, "c1"]
+         |  ]
+         |}""".stripMargin
+
+    val response: String = `given`
+      .body(request)
+    .when()
+      .post()
+    .`then`
+      .statusCode(HttpStatus.SC_OK)
+      .contentType(JSON)
+      .extract()
+      .body()
+      .asString()
+
+    assertThatJson(response).isEqualTo(
+      s"""{
+         |  "sessionState":"${SESSION_STATE.value}",
+         |  "methodResponses": [
+         |    ["error", {
+         |      "type": "unknownMethod",
+         |      "description": "Missing capability(ies): com:linagora:params:long:lived:token"
+         |    },"c1"]
+         |  ]
+         |}""".stripMargin)
+  }
+
+  @Test
+  def setDestroyShouldReturnFailWhenMissingAllCapabilities(): Unit = {
+    val request: String =
+      s"""{
+         |  "using": [],
+         |  "methodCalls": [
+         |    ["LongLivedToken/set", {
+         |      "accountId": "$ACCOUNT_ID",
+         |      "destroy": ["${LongLivedTokenId.generate.value}"]
+         |    }, "c1"]
+         |  ]
+         |}""".stripMargin
+
+    val response: String = `given`
+      .body(request)
+    .when()
+      .post()
+    .`then`
+      .statusCode(HttpStatus.SC_OK)
+      .contentType(JSON)
+      .extract()
+      .body()
+      .asString()
+
+    assertThatJson(response).isEqualTo(
+      s"""{
+         |  "sessionState":"${SESSION_STATE.value}",
+         |  "methodResponses": [
+         |    ["error", {
+         |      "type": "unknownMethod",
+         |      "description": "Missing capability(ies): urn:ietf:params:jmap:core, com:linagora:params:long:lived:token"
+         |    },"c1"]
+         |  ]
+         |}""".stripMargin)
+  }
+
+  @Test
+  def setDestroyShouldFailWhenWrongLongLivedTokenId(): Unit = {
+    val request: String =
+      s"""{
+         |  "using": ["urn:ietf:params:jmap:core", "com:linagora:params:long:lived:token"],
+         |  "methodCalls": [
+         |    ["LongLivedToken/set", {
+         |      "accountId": "$ACCOUNT_ID",
+         |      "destroy": ["wrong@id"]
+         |    }, "c1"]
+         |  ]
+         |}""".stripMargin
+
+    val response: String = `given`
+      .body(request)
+    .when()
+      .post()
+    .`then`
+      .statusCode(HttpStatus.SC_OK)
+      .contentType(JSON)
+      .extract()
+      .body()
+      .asString()
+
+    assertThatJson(response)
+      .inPath("$.methodResponses[0][1]")
+      .isEqualTo(
+        """
+           |{
+           |  "type": "invalidArguments",
+           |  "description": "{\"errors\":[{\"path\":\"obj.destroy[0]\",\"messages\":[\"LongLivedTokenId does not match Id constraints: Predicate failed: 'wrong@id' contains some invalid characters. Should be [#a-zA-Z0-9-_] and no longer than 255 chars.\"]}]}"
+           |}""".stripMargin)
+  }
+
+  @Test
+  def setDestroyShouldSucceedWhenUnknownLongLivedTokenId(): Unit = {
```

Comment:
That one I hesitated a while... 
* if you look at the core specs of jmap, if you try to destroy something that doesn't exist, it should return not found errror
* however that would imply to do extra reads with Cassandra and maybe we want to be more efficient on our custom specs...

After looking again at Keystore/set destroy that seems to return it as a success even for key ids not existing in our db, I thought was legit to apply point number 2 here. But tell me if you think otherwise :)